providers/salesloft/metadata/schemas.json

```diff
@@ -538,8 +550,8 @@
         },
         "external_configurations": {
           "displayName": "Team Configurations",
-          "path": "/external_configurations",
```

Comment:
i'm not going to comment on the rest of the objects in this file, but the object names should all use `/` if they exist in the path, not `_`